### DISCUSSION TITLE
属性选择界面控制

### DISCUSSION THREAD

**Discussion by 0kk470** [jynew/jynew#1299] [Ideas]
**功能描述**

开局属性随机

**UI导航路径**

无需导航路径，仅需要确认和取消。
需要支持键盘输入"baberuth"指令作弊获得全属性。
需要支持手柄输入"上上下下左右左右BABA"指令获得全属性。
![image](https://user-images.githubusercontent.com/25216715/199191985-6d83038b-9402-4499-9109-8296d8665f1f.png)

**Comment by 0kk470** [comment]
Done.

**Comment by MephistoSatanDevil** [comment]
作弊码是“baberuth”，不是“babyruth”。
另外确定取消按钮不宜变动，把手柄作弊码改成"↑↑↓↓←→←→XY"和"↑↑↓↓←→←→■▲"即可。
